openquake/hazardlib/calc/gmf.py

```diff
@@ -203,14 +203,15 @@ def compute(self, gsim, num_events, mean_stds):
             (len(self.imts), len(self.ctx.sids), num_events), F32)
         sig = numpy.zeros((M, num_events), F32)  # same for all events
         eps = numpy.zeros((M, num_events), F32)  # not the same
-        numpy.random.seed(self.seed)
+        rng = numpy.random.default_rng(self.seed)
```

Comment:
There's a similar line in conditioned_gmfs.py. Should we change that too?

Reply:
`conditioned_gmfs.py` is already good. I am changing the old code to avoid setting the global random seed generator, consistently with `conditioned_gmfs.py`.